main.js

```diff
@@ -140,7 +141,8 @@ async function main() {
         }
 
         if (!runID) {
-            throw new Error("no matching workflow run found with any artifacts?")
+            setExitMessage(ifNoArtifactFound, "no matching workflow run found with any artifacts?")
+            return
```

Comment:
```suggestion
            return exitWithMessage(ifNoArtifactFound, "no matching workflow run found with any artifacts?")
```

Could we do it like this?